docs/1.guide/3.websocket.md

```diff
@@ -103,7 +103,7 @@ As an alternative to WebSockets, you can use [Server-sent events](https://develo
 
 ### Example
 
-Create an SSE handler in `routes/sse.ts` (or `server/routes/api/sse.ts` for Nuxt).
+Create an SSE handler in `routes/sse.ts` (or `server/routes/sse.ts` for Nuxt).
```

Comment:
I'd say the other way around, it should be in `server/api/sse.ts` so it can live as API route 🤔

Reply:
The whole page uses `routes`, that's why I changed it in this place only.